docs/multi_file_content_management.md

```diff
@@ -0,0 +1,25 @@
+# Multi-file content management
+
+Multi-file content management is a set of functionalities that enables you to upload, move and delete multiple binary files in a bulk.
+
+### Multi-file upload
+
+To add multiple items select 'Upload sub-items' button from 'Sub-items' top bar and choose all items you want to upload.
```

Comment:
`**Upload sub-items**` <- bold or nothing, but we don't use quotation marks for UI elements elsewhere is user guide (also applies below).